pkg/docker/docker.go

```diff
@@ -619,19 +619,22 @@ func LoginToRegistry(address string, username string, password string) *DockerEr
 	// Pipe the password via stdin.
 	cmd := exec.Command("docker", "login", "--username", username, "--password-stdin", address)
 	stdin, err := cmd.StdinPipe()
+	textDockerLoginFailed := fmt.Sprintf("Docker login to %s as %s failed. Address, username or password is incorrect", address, username)
```

Comment:
Guess this can't go in the `docker_error.go` file as you want to pass in the username and address?

Reply:
That's why it's in the function yes